ethereum/truffle.js

```diff
@@ -4,6 +4,14 @@ module.exports = {
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
   networks: {
+    dev: {
+      host: "172.31.42.208",
```

Comment:
Is this static?

Reply:
Yes, for now. Ideally we'll have to configure this based on the environment but I don't have a good solution for that yet.